<launch>
  <!-- Define arguments: -->
  <!--
  <arg name="use_gripper" default="true"  doc="Should the gripper be used?" />  # example
  -->

  <!-- Define parameters: -->
  <!--
  <param name="param_name"  value="param_value" />  # example
  <param name="m_ee" type="double" value="0.76" if="$(arg use_gripper)" />  # example
  -->

  <!-- Load configuations into parameter server -->
  <rosparam file="$(find irobman_project)/config/general_config.yaml" command="load" />
  <rosparam file="$(find irobman_project)/config/tower1.yaml" command="load" />
  
  <!-- Launch nodes -->
  <node name="motion_planner" pkg="irobman_project" type="motion_planner.py" output="screen" />
  <arg name="node_start_delay" default="1.0" /> 
  <node name="state_machine" pkg="irobman_project" type="state_machine.py" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
</launch>
